tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -580,6 +580,9 @@ void SpirvEmitter::HandleTranslationUnit(ASTContext &context) {
 
   // The entry function is the seed of the queue.
   for (auto *decl : tu->decls()) {
+    if (context.getDiagnostics().hasErrorOccurred())
```

Comment:
nit: why don't we put it at the end of for-loop?

Reply:
done